content/influxdb/cloud/reference/release-notes/influx-cli.md

```diff
@@ -8,6 +8,20 @@ menu:
     name: influx CLI 
 ---
 
+## v2.1.1 [unreleased]
+
+## Go Version 
+
+This release upgrades the project to `go` version 1.17.
+
+## Bug Fixes 
+
+- Fix shell completion for top-level `influx` commands.
+- Make global `--http-debug` flag visible in help text.
```

Comment:
From an end-user viewpoint, does it make sense to say "Add global `--http-debug` flag? Or do we want to call out that it was there before, but just hidden? 

Notice we have documented this flag in some places already and the PR says it's a backport. Where did we backport from and/or is this important?

Reply:
It was backported from a merge to the `master` branch in the CLI repo.

I've told a few users to try `--http-debug` for debugging, but most probably don't know it exists. The current wording is most accurate but I wouldn't object to rephrasing it as a new CLI feature

Reply:
thanks, @danxmoran - @mluu12, we can leave text as is. I see you added a description to the main cli package, so that works